<launch>
	<!-- turtlebot_teleop_key already has its own built in velocity smoother -->
	<group ns="robot1">
		<node name="turtlebot_teleop_keyboard" output="screen" pkg="turtlebot_teleop" type="turtlebot_teleop_key">
			<param name="scale_linear" type="double" value="0.5"/>
			<param name="scale_angular" type="double" value="1.5"/>
			<remap from="turtlebot_teleop_keyboard/cmd_vel" to="cmd_vel_mux/input/teleop"/>
		</node>
	</group>
</launch>